docs_src/microservices/device/supported/device-usb-camera/walkthrough/custom-build.md

```diff
@@ -78,15 +78,34 @@ See the examples at `cmd/res/devices`
     When a new device is created in Core Metadata, a callback function of the device service will be called to add the device card name and serial number to protocol properties for identification purposes. These two pieces of information are obtained through `V4L2` API and `udev` utility.
 
 ## Configurable RTSP server hostname and port
-The hostname and port of the RTSP server can be configured in the `Driver` section of the `device-usb-camera/cmd/res/configuration.yaml` file. The default values can be used for this guide.
+The hostname and port of the RTSP server can be configured in the `Driver` section of the `device-usb-camera/cmd/res/configuration.yaml` file. The default values can be used for this guide. The `RtspAuthenticationServer` value indicates the hostname and port on which the authentication server will run. If this value is changed, you will have to also change the [mediamtx configuration]() 
```

Comment:
I would change the `RtspAuthenticationServer` part to be worded a little different to better express it:

> The `RtspAuthenticationServer` value indicates the internal hostname and port on which the device service will listen for RTSP authentication requests on. If this value is changed, you will have to also change the  [mediamtx configuration](...) to point to the new hostname/port as well.